extensions/Worker.Extensions.Kafka/src/Worker.Extensions.Kafka.csproj

```diff
@@ -21,7 +21,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <WebJobsExtension Include="Microsoft.Azure.WebJobs.Extensions.Kafka" Version="4.1.0" />
+    <WebJobsExtension Include="Microsoft.Azure.WebJobs.Extensions.Kafka" Version="4.1.1" />
```

Comment:
Can you bump line 9 as well to 4.1.2? `<VersionPrefix>4.1.2</VersionPrefix>`

The versions don't have to match - for example, [event hubs](https://github.com/Azure/azure-functions-dotnet-worker/blob/main/extensions/Worker.Extensions.EventHubs/src/Worker.Extensions.EventHubs.csproj) is  6.4.0 but uses the webjobs 6.5.1 package.